SSSOM.md

```diff
@@ -491,3 +492,77 @@ Not part of spec, clean up.
 *  explicitly declare the metadata elements as equivalent to properties?
 * What about mapping to Literals in general? Is this in-spec?
+
+<a name="minimum"></a>
+
+### Minimum Metadata Recommendation
+
+Current mappings are extremely hard to use for data integration, because they are:
+
+- non-transparently imprecise
+- non-transparently incomplete
+- inaccurate
+- unFAIR
+
+In principle, to reach full integration through mappings, you will have to cross-map all ontologies, or semantic
+spaces, which means if you have N "spaces", you have N x N (-1) mappings. To mitigate the explosion
+of mappings, we have to be able to cross-walk:
+- multi-hop forward walks `{ O1:A->O2:A, O2:A->O3:A } --> {O1:A->O3:A}`
+- walk-backs `{ O1:A->O2:A } --> {O2:A->O1:A}`
+- combinations `{ O1:A->O2:A, O2:A->O3:A, O4:A->O3:A } --> {O1:A->O4:A}`
+
+To enable cross-walking, we propose the following 5 star system for mapping metadata. 
+
+1. _1-star mappings_ fulfil the following criteria:
+   - record subject id, object id and mapping precision
+   - using qualified names (either URIs or CURIEs + curie maps) for subject id and object id
+   - using a standard file format (JSON, XML, CSV, TSV)
+   - made available in a public space
+   - _optional_: record the subject and object labels to make it easier for humans to read the file
+2. _2-star mappings_ fulfil all the criteria for 1-star mappings and furthermore
+   - made available in a public version control system with an issue tracker
+   - record the semantic predicate explicitly
+   - using qualified names for the semantic predicate (i.e. owl:equivalentClass, skos:exactMatch)
+   - record a confidence value for the mapping between 0 and 1.
+   - record an open license for the use of the mapping set
+3. _3-star mappings_ fulfil all the criteria for 2-star mappings and furthermore 
+   - are stored in SSSOM format
+   - record the following additional metadata: 
+     - `match_type`(s) (Lexical, Logical match, HumanCurated etc)
+     - `date` of the mapping
+     - `creator_id`
+     - `subject_source`
+     - `object_source`
+     - `subject_source_version`
+     - `object_source_version`
+     - `mapping_tool` if the mapping was automatically computed using a tool
+4. _4-star mappings_ fulfil all the criteria for 3-star mappings and furthermore
+   - Register there mapping at a mapping commons
+   - record the following additional metadata:
+     - `mapping_set_id`
+     - `mapping_set_description`
+     - `mapping_set_version`
+     - `mapping_provider` (if the mapping is not original, i.e. it is not derived from another source)
```

Comment:
the rules for what are valid mapping providers probably need to be fleshed out. Are these OBO prefixes/other stuff that might appear in the Bioregistry or FAIRSharing?

Reply:
I think if the mapping set is original, i.e. not derived from some source, than we would do users a disservice to overly specify the mapping_provider. I will raise this issue at the workshop - I am sure this is a super standard provenance problem and I will try to get a provenance person there as well.

Reply:
ORCID ID of the main responsible person? or wikidata/ROR ID for an organization if responsibility is _truly_ decentralized?

Reply:
I think both should be permissible, yes.

Reply:
So in general, we should ask the `mapping_provider` be a valid CURIE and give a warning if one of the preferred prefixes are not used

Reply:
What if the mapping provider is a database? it seems a bit odd then to require a CURIE rather than a URL.. This is not the same as `subject_id` imo..

Reply:
use a [fairsharing](http://bioregistry.io/registry/fairsharing) CURIE corresponding to the database :)

You can't trust URLs don't go away over time due to databases shutting down from funding drying up/people moving on, but an entry in an external database like FAIRSharing, WikiData, or even a Bioregistry CURIE could have much more longevity

Reply:
I agree this would be better.. However, how practical for all cases I don't know.. Lets discuss it for a minute at the workshop.